src/Apps/__tests__/Fixtures/Sale.ts

```diff
@@ -0,0 +1,11 @@
+import { routes_AuctionQueryResponse } from "__generated__/routes_AuctionQuery.graphql"
+
+export const SaleAuction: routes_AuctionQueryResponse["sale"] = {
+  " $fragmentRefs": undefined,
```

Comment:
@damassi How does this look? :eyes: 

cc/ @dleve123

Reply:
Not sure of the specifics, but there are other examples  of typed fixtures in reaction that can be used as a reference.